src/main/java/core/basesyntax/figures/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,28 @@
+package core.basesyntax.figures;
+
+public class IsoscelesTrapezoid extends Figure {
+    private final double baseUp;
+    private final double baseBottom;
+    private final double height;
+
+    public IsoscelesTrapezoid(String color, double baseUp, double baseBottom, double height) {
+        super(color);
+        this.baseUp = baseUp;
+        this.baseBottom = baseBottom;
+        this.height = height;
+        calcArea();
```

Comment:
Don`t send it to abstract class use it here

Reply:
I am sending only color to abstract`s  class constructor. You mean don`t send color to abstract class ?

Reply:
I mean that don`t invoke calcArea() in constructor and your method store result in field area in your abstract class